src/components/PatientProfilePage/PatientProfilePage.tsx

```diff
@@ -178,6 +307,9 @@ const PatientProfilePage = ({
       </div>
       <div className="event-form">
         <ValidatorForm onSubmit={handleComplete}>
+          <Typography variant="h5" style={{ marginBottom: 24 }}>
```

Comment:
I've a noticed several occurrences of non px values, do we want to be consistent?